docs/api/effector-solid/index.md

```diff
@@ -0,0 +1,19 @@
+---
+id: effector-solid
+title: API Reference
+---
+
+## Reactive helpers
+
+- [useStore(store)](./useStore.md)
```

Comment:
As I understand now, `effector-react` is moving to simple hook like `useUnit`. Shall we switch this package to the same approach before release?

I think, it'll be harder to do such moves after release.

Reply:
https://github.com/effector/effector/pull/685#discussion_r875774633

I got it 🤣